docs/operate/config-reference.mdx

```diff
@@ -428,7 +428,7 @@ export HYP_CHAINS_${CHAIN_NAME}_SIGNER_TYPE="hexKey"
 
 **Requires:** `chains.<chain_name>.signer.type = "hexKey" | undefined`
 
-**Optional:** No (iff requirements are met)
+**Optional:** No (if requirements are met)
```

Comment:
"iff" means "if and only if," which is more precise in this context and with the other ones on the same file

Reply:
Thank you!
Completed ✅